app/models/audio_file.rb

```diff
@@ -32,7 +32,7 @@ class AudioFile < BaseModel
       self.status ||= UPLOADED
     end
 
-    if !account && story
+    if !account_id && story.try(:account_id)
```

Comment:
Was seeing a testing bug where the `account_id` would change, but `story.account` was nil (and refused to auto-load).  Odd test failures!